tests/src/test/java/alluxio/server/ft/MultiWorkerIntegrationTest.java

```diff
@@ -172,6 +172,7 @@ public void positionReadRecoverFromLostWorker() throws Exception {
       byte[] buf = new byte[length];
       replicateFileBlocks(filePath);
       mResource.get().getWorkerProcess().stop();
+      Thread.sleep(500);
```

Comment:
What is the issue that is solved by a sleep? Can we wait for some other condition instead of sleeping?